### PR TITLE
zh-cn: update the translation of `TypedArray.every()`

### PR DESCRIPTION
**Pull request by yin1999**
### Description

update the translation of `TypedArray.every()`

### CODE REVIEW

File: files/zh-cn/web/javascript/reference/global_objects/typedarray/every/index.md [hunk 1]
```diff
@@ -17,35 +21,29 @@ every(callbackFn, thisArg)
...
+  - : 用于测试类型化数组中每一个元素的函数。其应该返回[真值](/zh-CN/docs/Glossary/Truthy)来表示元素通过了测试，否则返回[假值](/zh-CN/docs/Glossary/Falsy)。函数会以下面的参数被调用：
+    - `element`
+      - : 类型化数组中要处理的当前元素。
     - `index`
       - : 要处理的当前元素在类型化数组中的下标
     - `array`
-      - : every 在其上调用的类型化数组
-- `thisArg`
-  - : 可选，执行 `callback` 时的 `this` 值。
+      - : 调用 `every()` 的类型类型化数组。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/typedarray/every/index.md:30]:
```suggestion
      - : 调用 `every()` 的类型化数组。
```

File: files/zh-cn/web/javascript/reference/global_objects/typedarray/every/index.md [hunk 2]
```diff
@@ -17,35 +21,29 @@ every(callbackFn, thisArg)
...
+- `thisArg` {{optional_inline}}
+  - : 该值会作为执行 `callbackFn` 时的 `this` 值。参见[迭代方法](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Array#迭代方法)。
 
 ### 返回值
 
-如果函数对数组每个元素返回 {{Glossary("truthy")}} ，则为**`true`**，否则为 **`false`**。
+除非 `callbackFn` 对类型化数组中的某个元素返回了{{Glossary("falsy", "假")}} 值（在这种情况下，立即返回 `false`），否则返回 `true`。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/typedarray/every/index.md:36]:
```suggestion
返回 `true`，除非 `callbackFn` 对类型化数组中的某个元素返回了{{Glossary("falsy", "假")}}值（在这种情况下，立即返回 `false`）。
```

File: files/zh-cn/web/javascript/reference/global_objects/typedarray/every/index.md [hunk 3]
```diff
@@ -17,35 +21,29 @@ every(callbackFn, thisArg)
...
 ## 描述
 
-`every` 方法为类型化数组中的每个元素执行一次 `callback` 函数，直到它找到一个使 `callback` 返回 _false_（表示可转换为布尔值 false 的值）的元素。如果发现了一个这样的元素，`every` 方法将会立即返回 `false`。否则，`callback` 为每一个元素返回 `true`，`every` 就会返回 `true`。`callback` 只会为那些已经被赋值的索引调用。不会为那些被删除或从来没被赋值的索引调用。
-
-`callback` 以三个参数调用：元素的值，元素索引，以及要遍历的数组对象。
-
-如果将 `thisArg` 参数提供给 `every`，它会在调用时传递给 `callback`，作为它的 `this` 值。否则，会传递 `undefined` 作为它的 `this` 值。`callback` 最终观测到的 `this` 值由[用于决定函数可见的 `this` 值的一般规则](/zh-CN/docs/Web/JavaScript/Reference/Operators/this)来决定。
-
-`every` 不修改在其上调用的类型化数组。
+参见 {{jsxref("Array.prototype.every()")}} 以了解详情。该方法不是通用，只能在类型化数组实例上调用。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/typedarray/every/index.md:40]:
```suggestion
参见 {{jsxref("Array.prototype.every()")}} 以了解详情。该方法不是通用的，只能在类型化数组实例上调用。
```

**Review by jasonren0403 — APPROVED** [review]
LGTM, Thank you @yin1999 👍

**Pull request merged**